insights/parsers/rear_conf.py

```diff
@@ -0,0 +1,37 @@
+"""
+ReaRLocalConf - File /etc/rear/local.conf
+=========================================
+"""
+
+from insights.core import TextFileOutput
+from insights.core.exceptions import SkipComponent
+from insights.core.plugins import parser
+from insights.parsers import get_active_lines
+from insights.specs import Specs
+
+
+@parser(Specs.rear_local_conf)
+class ReaRLocalConf(TextFileOutput):
+    """
+    Parses content of "/etc/rear/local.conf".
+
+    Typical content of "/etc/rear/local.conf" is::
+
+        BACKUP_RESTORE_MOVE_AWAY_FILES=( /boot/grub/grubenv /boot/grub2/grubenv )
+
+
+    Examples:
+        >>> type(local_conf)
+        <class 'insights.parsers.rear_conf.ReaRLocalConf'>
+        >>> local_conf.data[0] == 'BACKUP_RESTORE_MOVE_AWAY_FILES=( /boot/grub/grubenv /boot/grub2/grubenv )'
+        True
+    """
+
+    def parse_content(self, content):
+        _content = get_active_lines(content)
```

Comment:
`self.data` is not recommended to be used as it's too common to be an attr name.

Reply:
as `TextFileOutput` is used as the base, you can just `pass` the parser unless you have new requires other than the base for this spec.